src/main/java/com/googleinterns/zoomtube/data/Comment.java

```diff
@@ -67,11 +68,10 @@ public static Comment fromEntity(Entity entity) {
   public abstract Date timestamp();
 
   /**
-   * Returns the comment's author. This is the email of the user who posted the
-   * comment.
+   * Returns the comment's author. This is the {@link com.google.appengine.api.users.User}
+   * who posted the comment.
```

Comment:
The second sentence can be removed. No need to specify that is it com.google.appengine.api.users.User given you can look at the method declaration to see 'User'

Reply:
Fixed